src/EA.Weee.DataAccess/StoredProcedure/TransferNoteData.cs

```diff
@@ -23,7 +23,7 @@ public class TransferNoteData
 
         public string EvidenceNoteReference { get; set; }
 
-        public DateTime EvidenceNoteApprovalDate { get; set; }
+        public DateTime? EvidenceNoteApprovalDate { get; set; }  // sql query moanied about this not being nullable
```

Comment:
@GuidoWorkDev  I cant see why this would be the case may have to look at what data causes this as all evidence notes for transfer notes should have an approval date. Perhaps funny data. In any case this should really be as it was.

Can you make sure you are complete up to date with the database updates.

Reply:
@GuidoWorkDev will need to find out why this was needed... Will pick up on Tuesday

Reply:
In the case on my dataset there were some nulls in the EvidenceNoteApprovalDate column which raised errors in the handler.  it can fix my dataset (remove the NULL values) and I will rollback to original type.